lib/playground.dart

```diff
@@ -199,6 +202,15 @@ class Playground implements GistContainer, GistController {
         blankGist.getFile('styles.css').content = pad.css;
         editableGist.setBackingGist(blankGist);
       });
+    } else if (url.hasQuery && url.queryParameters['source'] != null) {
+      Future<UuidContainer> futureGistId = dartSupportServices
+          .retrieveGist(id: url.queryParameters['source']);
+      await futureGistId.then((UuidContainer gistId) {
```

Comment:
And, generally, I'm not sure why this method is declared to return a Future when we not returning values from it